cmp/ag-grid/AgGridModel.ts

```diff
@@ -347,7 +349,8 @@ export class AgGridModel extends HoistModel {
 
             // finally apply sorts from state to secondary columns
             secondaryColumnState.forEach(state => {
-                const col = colApi.getSecondaryPivotColumn(state.colId[0], state.colId[1]);
+                // TODO: Check this -- was very off
```

Comment:
Is this TODO still relevant?

Reply:
yes -- just changed this code without running it -- need to test pivot grids still

Reply:
restored the code -- this was already broken, and not looking to change runtime behavior in this change.  Will file a ticket